src/v3/test/integration/__snapshots__/authenticator-enroll-data-phone.test.tsx.snap

```diff
@@ -1470,7 +1470,7 @@ exports[`authenticator-enroll-data-phone should render form 1`] = `
                             <span
                               class="ods-4o5zYQ ods-nqMobL ods-5fvoER ods-6rDd3P ods-7KlglL ods-6MIfC2 ods-14pjkk ods-5IIKpe ods-6iOjmZ"
                             >
-                              Phone number
```

Comment:
looks like this one is lost during transformation

Reply:
Fixed, this is caused because the Field transformer somehow doesnt include the phone label (not sure why because it works for things like credentials.passcode, etc, but I added it manually. Going to scan through the other views to ensure this didnt happen anywhere else.